hiveos/h-manifest.conf

```diff
@@ -0,0 +1,26 @@
+####################################################################################
+###
+### kaspa-miner
+### https://github.com/tmrlvi/kaspa-miner/releases
+###
+### Hive integration: Merlin
+###
+####################################################################################
+
+# The name of the miner
+CUSTOM_NAME=kaspa-miner
+
+# Optional version of your custom miner package
+CUSTOM_VERSION=0.1.3
```

Comment:
Is there way to take the version from the `toml` file? Or is hiveos a release with binaries?